builder/azure/arm/config.go

```diff
@@ -414,6 +414,8 @@ type Config struct {
 	// standard or premium. The default value is standard.
 	BuildKeyVaultSKU string `mapstructure:"build_key_vault_sku"`
 
+	SkipCreateBuildKeyVault bool `mapstructure:"skip_create_build_key_vault" required:"false"`
```

Comment:
We should add a comment describing this field here and then run `make generate`, this will generate docs for this field, I'm happy to write that but I'd like to have a better understanding of the userdata/custom_script so I can write it with those in mind since they're required to make this field work I believe